convert_fendl.py

```diff
@@ -41,90 +41,289 @@ class CustomFormatter(argparse.ArgumentDefaultsHelpFormatter,
                     default='earliest', help="Output HDF5 versioning. Use "
                     "'earliest' for backwards compatibility or 'latest' for "
                     "performance")
-parser.add_argument('-r', '--release', choices=['3.1a', '3.1d'],
+parser.add_argument('-r', '--release', choices=['3.1a', '3.1d', '3.0', '2.1'],
                     default='3.1d', help="The nuclear data library release version. "
-                    "The currently supported options are 3.1a and 3.1d")
+                    "The currently supported options are 3.1d, 3.1a, 3.0 and "
+                    "2.1")
+parser.add_argument('-p', '--particles', choices=['neutron', 'photon'], 
+                    nargs='+', default=['neutron', 'photon'], 
+                    help="Incident particles to include")
 parser.set_defaults(download=True, extract=True)
 args = parser.parse_args()
 
 
 # this could be added as an argument to allow different libraries to be downloaded
 library_name = 'fendl'
-ace_files_dir = Path('-'.join([library_name, args.release, 'ace']))
+cwd = Path.cwd()  
+
+ace_files_dir = cwd.joinpath('-'.join([library_name, args.release, 'ace']))
+endf_files_dir = cwd.joinpath('-'.join([library_name, args.release, 'endf']))
+
+download_path = cwd.joinpath('-'.join([library_name, args.release, 'download']))
 # the destination is decided after the release is know to avoid putting the release in a folder with a misleading name
 if args.destination is None:
     args.destination = Path('-'.join([library_name, args.release, 'hdf5']))
 
 # This dictionary contains all the unique information about each release. This can be exstened to accommodated new releases
```

Comment:
Although this comment has not been changed in this PR, we could take the opportunity to fix the mangled English.